### ISSUE TITLE
global: fix build failure

### ISSUE THREAD

**Issue by ntarocco** [inveniosoftware/invenio-iiif#31]
https://travis-ci.org/inveniosoftware/invenio-iiif/jobs/656677007
```
/home/travis/virtualenv/python2.7.15/lib/python2.7/site-packages/sphinx/util/nodes.py:94: FutureWarning: 
   The iterable returned by Node.traverse()
   will become an iterator instead of a list in Docutils > 0.16.
  for classifier in reversed(node.parent.traverse(nodes.classifier)):
Warning, treated as error:
autodoc: failed to import module u'tasks' from module u'invenio_iiif'; the following exception was raised:
cannot import name LocalProxy
The command "./run-tests.sh" exited with 2.
```

Fix also deprecation warning in Travis config:
https://travis-ci.org/inveniosoftware/invenio-iiif/jobs/656677007/config

**Comment by ppanero** [comment]
The above error is on Python 2.7 builds (Not supported anymore).

Python 3 builds break with:

```
    from flask import current_app

>   from werkzeug import cached_property

E   ImportError: cannot import name 'cached_property'
```

**Comment by slint** [comment]
Closed via #33
